### ISSUE TITLE
Use PICRUSt2 to visualize pathways that differ between the numerator and denominator of a log-ratio

### ISSUE THREAD

**Issue by fedarko** [biocore/qurro#210]
Thanks to @rhmills for the cool idea. As I understand it, what this would entail is:

- Convert the BIOM table to a presence/absence table (or just use the representative sequences file)
- Run this sequence data through PICRUSt2
- This produces a `path_abun_unstrat` file, which describes pathways' predicted "relative abundance" for each feature
- We could store this information in the Qurro visualization, and then show a plot of the most "differentially abundant" pathways for the numerator and denominator of your pathways. There are a couple of approaches we could take here.

Basically, this would provide a functional context to the interpretation of feature log-ratios, in addition to the feature rankings and sample abundances that are currently shown.
